lib/screens/widgets/stufenwechselInfo.widget.dart

```diff
@@ -90,73 +88,53 @@ class _StufenwechselInfoState extends State<StufenwechselInfo> {
-      padding: const EdgeInsets.all(16.0),
-      child: Row(
-        children: [
-          // Bereich für die Stufenbilder
-          SizedBox(
-            width: 50.0,
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.stretch,
-              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-              children: List.generate(stufenBilder.length, (index) {
-                return GestureDetector(
-                  onTap: () {
-                    _stufeAendern(index);
-                  },
-                  child: Container(
-                    width: 50.0,
-                    height: 50.0,
-                    decoration: BoxDecoration(
-                      shape: BoxShape.circle,
-                      color: ausgewaehlteStufe == index
-                          ? Colors.blue
-                          : Colors.grey,
-                    ),
-                    child: Center(
-                      child: Image.asset(
-                        stufenBilder[index],
-                        width: 30.0,
-                        height: 30.0,
-                      ),
-                    ),
+    return Column(
+      children: [
+        // Bereich für die Stufenbilder
+        Wrap(
+          spacing: 5,
+          children: [
+            for (final stufe in Stufe.stufenWithoutLeiter)
+              ChoiceChip(
+                selected: stufe == ausgewaehlteStufe,
+                onSelected: (selected) {
+                  if (selected) _stufeAendern(stufe);
+                },
+                label: Text(stufe.shortDisplay),
+                avatar: CircleAvatar(
+                  backgroundColor: Colors.transparent,
+                  child: Image.asset(
+                    stufe.imagePath!,
                   ),
-                );
-              }),
-            ),
-          ),
-          Container(
-            margin: const EdgeInsets.only(left: 16.0, right: 5),
-            width: 1.0,
-            color: Colors.grey,
-            height: availableHeight, // Höhe anpassen, abhängig von Ihrer Liste
-          ),
-          // Bereich für die Liste der Namen
-          Expanded(
-            child: Align(
-              alignment: Alignment.topCenter,
-              child: SingleChildScrollView(
-                scrollDirection: Axis.vertical,
-                child: aktuelleTabellenZeilen.isNotEmpty
-                    ? DataTable(
-                        columns: const [
-                          DataColumn(label: Text('Name')),
-                          DataColumn(label: Text('Wechsel'))
-                        ],
-                        rows: aktuelleTabellenZeilen,
-                      )
-                    : const Text('Kein Wechsel zum angegebenen Datum'),
+                ),
               ),
+          ],
+        ),
+        // // Bereich für die Liste der Namen
+        if (aktuelleTabellenZeilen.isNotEmpty)
+          Card(
+            child: DataTable(
```

Comment:
Die Card sollte eine Fixe breite beklommen. Aktuell schrumpft sie, wenn die Namen in einer anderen Stufe weniger Platz benötigen
<img width="265" alt="Bildschirmfoto 2024-03-29 um 01 05 33" src="https://github.com/digital-scouts/dpsg-nami-app/assets/26873867/a81a9831-757d-4662-8240-ba705ef5da48">
<img width="265" alt="Bildschirmfoto 2024-03-29 um 01 05 27" src="https://github.com/digital-scouts/dpsg-nami-app/assets/26873867/795206cd-dd4a-4f63-a598-3b0ccabc3c50">